packages/reakit/src/Popover/__tests__/PopoverArrow-test.tsx

```diff
@@ -0,0 +1,250 @@
+import * as React from "react";
+import { render } from "react-testing-library";
+import PopoverArrow from "../PopoverArrow";
+
+test("html attrs", () => {
+  const { getByTestId } = render(
+    <PopoverArrow data-testid="test" aria-label="test" />
+  );
+  expect(getByTestId("test")).toHaveAttribute("aria-label", "test");
+});
+
+test("fillColor", () => {
+  const { container } = render(<PopoverArrow fillColor="red" />);
+  expect(container.firstChild).toMatchInlineSnapshot(`
+.c1 {
```

Comment:
@diegohaz what do you think about putting these long snapshots into their files (i mean, using toMatchSnapshot instead) ?

Reply:
I think that's a good idea in this sort of case, when there are multiple snapshots for style changes

Reply:
I agree that long snapshots should be using `toMatchSnapshot`. For the other cases, we should use `toMatchInlineSnapshot` whenever possible.

Reply:
Go for it @amogower